Core/include/Acts/Fitter/detail/VoidKalmanComponents.hpp

```diff
@@ -59,8 +59,7 @@ struct VoidMeasurementCalibrator {
                               source_link_t>,
         "For DefaultMeasurementCalibrator, source link needs to implement "
         "dereference operator");
-
-    return *sl;
+    return *sourceLink;
```

Comment:
Does it warn about the argument name length?

Reply:
No, this is just clarification.